### PR TITLE
Improving astroplan overviews in docs

### PR DESCRIPTION
**Pull request by bmorris3**
In response to #75 (@cdeil)

### CODE REVIEW

**Comment by bmorris3** [comment]
This PR overlaps with #77 a bit, particularly in `docs/index.rst`. I'll wait until #77 is merged before revising this PR.

File: docs/index.rst [hunk 1]
```diff
@@ -14,28 +14,29 @@ observations.
 
 .. warning::
 
-    We're still in development, so the `astroplan` API may change.
-    `Astropy`_, `PyEphem`_ or `Skyfield`_ are good options if you're looking
-    for a more stable package.
+    astroplan is still in development, so the `astroplan` API may change
+    suddenly and without warning, like the weather. `PyEphem`_ and `Skyfield`_
```
Comment (cdeil) [inline docs/index.rst]: :+1: :smile:

**Comment by cdeil** [comment]
:+1:

**Comment by bmorris3** [comment]
This closes astropy/astroplan#75.

**Pull request merged**